src/app/shared/components/template/components/table/table.component.ts

```diff
@@ -0,0 +1,82 @@
+import {
+  AfterViewInit,
+  ChangeDetectorRef,
+  Component,
+  computed,
+  effect,
+  OnInit,
+  ViewChild,
+} from "@angular/core";
+import { TemplateBaseComponent } from "../base";
+import { FlowTypes } from "packages/data-models";
+import { getBooleanParamFromTemplateRow } from "src/app/shared/utils";
+import { AppDataService } from "src/app/shared/services/data/app-data.service";
+import { MatTableDataSource } from "@angular/material/table";
+import { MatSort } from "@angular/material/sort";
+
+interface ITableParams {
+  /** TEMPLATE PARAMETER: show_id. Display the data lists's ID column in the table. Default false */
+  showId: boolean;
+}
+
+@Component({
+  selector: "plh-table",
+  templateUrl: "./table.component.html",
+  styleUrls: ["./table.component.scss"],
+})
+export class TmplTableComponent extends TemplateBaseComponent implements AfterViewInit {
+  params = computed(() => this.getParams(this.parameterList()));
+  dataRows = computed(() => this.getDataRowsFromValue(this.value()));
```

Comment:
nit(blocking): the `getDataRows` from value function is async, so this computed will not behave as expected (no such thing as async computed). Instead it would be better to use an effect to respond to value changes and set the dataRows as a signal. I think it still kinda works in the frontend due to the additional cdr calls, but I'd consider that to be more of a bug than expected behaviour